engine/storage/snapshot/src/main/java/org/apache/cloudstack/storage/snapshot/SnapshotServiceImpl.java

```diff
@@ -373,7 +373,7 @@ protected Void deleteSnapshotCallback(AsyncCallbackDispatcher<SnapshotServiceImp
                 res = new SnapshotResult(context.snapshot, null);
             }
         } catch (Exception e) {
-            s_logger.debug("Failed to in deleteSnapshotCallback", e);
+            s_logger.error("An exception occurred while processing an event in delete snapshot callback.", e);
```

Comment:
```suggestion
            s_logger.error("An exception occurred while processing an event in delete snapshot callback.");
            s_logger.debug("Exception while processing an event in delete snapshot callback.", e);
```
stacktraces should not go in error level logging